lib/msal-browser/src/controllers/StandardController.ts

```diff
@@ -706,6 +678,7 @@ export class StandardController implements IController {
             }
 
             let result: Promise<void>;
+            window.addEventListener("beforeunload", handleUnload);
```

Comment:
We've avoided doing this in the past because it isn't supported on all browsers and its usage [breaks BFCache](https://developer.mozilla.org/en-US/docs/Web/API/Window/beforeunload_event#usage_notes)